src/js/cellAcl.js

```diff
@@ -108,7 +108,10 @@ cellAcl.prototype.implementCellACL = function (arrCellACLRolePrivilegeList) {
 	var objAce = null;
 	var objJAcl = new _pc.Acl();
 	var objPrincipal = new _pc.Principal();
-	objJAcl.setRequireSchemaAuthz("public");
+	var rolePrivList = this.getCellACLSettings();
+	if (rolePrivList.schemaAuthz != "null") {
```

Comment:
Should move logic to the function that uses schemaAuthz since it is already filtering out ""/null/undefined.

Reply:
corresponded.